WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1482,7 +1481,7 @@ public int getCount() {
 
     private int getMaximumThumbnailWidthForEditor() {
         if (mMaxThumbWidth == 0) {
-            mMaxThumbWidth = ImageUtils.getMaximumThumbnailWidthForEditor(this);
+            mMaxThumbWidth = org.wordpress.android.editor.MediaUtils.getMaximumThumbnailSizeForEditor(this);
```

Comment:
I was wondering, maybe we can rename this class from `MediaUtils` to a prefixed name like`EditorMediaUtils`, even when that would be redundant with the `org.wordpress.android.editor` path, but just to be able to make a clearer distinction between all the `MediaUtils` we are using in this file. We have:
- `import org.wordpress.android.util.MediaUtils;`
- `import org.wordpress.android.util.WPMediaUtils;`
- and `org.wordpress.android.editor.MediaUtils` used in this line here